README.md

```diff
@@ -78,16 +68,15 @@ npm run test
 
 ### Build
 
-Before you build your lens, you should update your lens metadata in `./SuperDuperLens/lens.json`.
-Note: "lensEventApiVersion" currently defaults to 1.
+Before you build your lens, you should update your lens project data in `./lens-name/package.json`.
```

Comment:
Instead of 
```update your lens project data in ./lens-name/package.json``` 

how about
```update your lens project data in the `package.json` file in the project's root directory```
?